### PR TITLE
NewClientWithJSONSlogger

### PR DESCRIPTION
**Pull request by itishrishikesh**
Please review the below changes included with this PR,
-- Added `NewClientWithJSONSlogger` to print logs in JSON format.
-- Some changes in the Readme file for the above function.
-- An example under cmd/examples for the above function.

### CODE REVIEW

**Comment by bhagatvansh** [comment]
LGTM!

File: client/README.md [hunk 1]
```diff
@@ -41,12 +41,12 @@ The options that are used for initializing the client are as follows:
...
-- **service_instance_name**: This is the name of the calling app's session. This value is supposed to indicate which instance among possibly multiple instances of a service sent a log message. For example, in case of the service being deployed within Kubernetes, it might indicate the service's pod's name. If the value is sent as a blank string, client will try to use the machine's hostname. If it fails to fetch the hostname, a random string will be used instead. 
+- **service_instance_name**: This is the name of the calling app's session. This value is supposed to indicate which instance among possibly multiple instances of a service sent a log message. For example, in case of the service being deployed within Kubernetes, it might indicate the service's pod's name. If the value is sent as a blank string, client will try to use the machine's hostname. If it fails to fetch the hostname, a random string will be used instead.
```
Comment (vaibhav-kaushal) [inline client/README.md:44]: Can we change `This is the name of the calling app's session` to `This is the name of the calling app's instance.`

**Review by vaibhav-kaushal — CHANGES_REQUESTED** [review]
Just one teeny tiny change required in the readme. Rest looks good.

File: client/README.md (hunk 1 above)
Comment (itishrishikesh) [reply client/README.md]: Done!

**Pull request merged**